### ISSUE TITLE
Content Security Policy on About page

### ISSUE THREAD

**Issue by NistorCristian** [status-im/status.im#387]
There seems to be a CSP issue on [About page](https://status.im/about/), on Active Swarms section. Because of this the images don't load.

There seems to be a CSP directive defined somewhere but not in hexo:
default-src 'self' *.status.im boards.greenhouse.io

Any ideas where this is defined @jakubgs ?

We're going to have to add:
*.githubusercontent.com
statusim.bamboohr.com

As an alternative I guess we could put a [meta tag in head](https://developers.google.com/web/fundamentals/security/csp/#the_meta_tag). But I want to make sure that this is not defined in another place first.

**Comment by jakubgs** [comment]
Yes, it's defined here:
https://github.com/status-im/infra-misc/blob/29f9d2c6469e39255bdcc69ccc1f6cb5cebd88da/files/site_headers.js#L10-L14
It works though CloudFlare workers, because we don't have any control over what headers GitHub pages return.

**Comment by jakubgs** [comment]
You want me to add:
```
*.githubusercontent.com
statusim.bamboohr.com
```
But to what? `script-src`? `style-src`? `default-src?

**Comment by NistorCristian** [comment]
Should work with default-src

**Comment by jakubgs** [comment]
Ah, I see, for `statusim.bamboohr.com` it's just images from:
```
https://statusim.bamboohr.com/employees/photos/*
```
And for `*.githubusercontent.com` it's:
```
https://avatars0.githubusercontent.com/u/*
```
So that should be `img-src`, in theory at least, those don't seem to have file extensions, though they should return `Content-Type` headers.

**Comment by NistorCristian** [comment]
Probably *.status.im needs to be added there too.

**Comment by jakubgs** [comment]
That's already in `default-src`.

**Comment by NistorCristian** [comment]
Yes but I think it needs to be in img-src too. Right now I see no image on status.im:

Refused to load the image 'https://status.im/img/favicon/apple-icon-57x57.png' because it violates the following Content Security Policy directive: "img-src *.githubusercontent.com statusim.bamboohr.com".

**Comment by jakubgs** [comment]
Oh, because it takes the narrower one by default, good point.

**Comment by jakubgs** [comment]
What it actually needs is a `'self'`.

**Comment by jakubgs** [comment]
Fixed - a601bf3 - deployed:
```
 $ curl -is https://status.im/about/ | grep content-security | tr ';' '\n' 
content-security-policy: default-src 'self' *.status.im boards.greenhouse.io
 style-src 'self' cdnjs.cloudflare.com fonts.googleapis.com stackpath.bootstrapcdn.com
 script-src 'self' 'unsafe-inline' unpkg.com code.jquery.com cdn.jsdelivr.net cdnjs.cloudflare.com stackpath.bootstrapcdn.com boards.greenhouse.io
 img-src 'self' *.githubusercontent.com statusim.bamboohr.com
 font-src 'self' fonts.gstatic.com fonts.googleapis.com
 frame-ancestors 'self' dash.cloudflare.com
```

**Comment by jakubgs** [comment]
Good catch, I knew I'd miss some things when introducing CSP, but it had to be done eventually.
